services/terrain/src/terrain/clients/apps/raw.clj

```diff
@@ -582,6 +582,27 @@
                :as               :stream
                :follow-redirects false}))
 
+(defn get-oauth-token-info
+  [api-name]
+  (client/get (apps-url "oauth" "token-info" api-name)
+              {:query-params     (secured-params)
+               :as               :stream
+               :follow-redirects false}))
+
+(defn get-oauth-redirect-uris
+  []
+  (client/get (apps-url "oauth" "redirect-uris")
+              {:query-params     (secured-params)
+               :as               :stream
```

Comment:
In general, I would agree with this. My only reservation for making the suggested change is that it reduces the consistency in the behavior of the functions in `terrain.clients.apps.raw` and also reduces the consistency in the implementation of the functions in `terrain.clients.apps`.